src/dcap_provider.cpp

```diff
@@ -67,7 +68,7 @@ static std::string cert_base_url = DEFAULT_CERT_URL;
 static char DEFAULT_CLIENT_ID[] = "production_client";
 static std::string prod_client_id = DEFAULT_CLIENT_ID;
 
-static char DEFAULT_COLLATARAL_VERSION[] = "v1";
+static char DEFAULT_COLLATARAL_VERSION[] = "v2";

```

Comment:
Can you please fix the spelling of collateral - sorry, stuff like that just bugs me :).

Reply:
Thanks for pointing it out :)